packages/example-getting-started/docs/7-controller.md

```diff
@@ -64,6 +64,21 @@ export class TodoController {
 }
```

Comment:
The code block would've been too much for me to take in if I was a new user. We should move the code block at the end of the docs and replace it with just the constructor and a single route so that it's easier to take in.

Reply:
We expect users to copy-paste this code block. I'll add a statement to that effect.

Reply:
I don't agree; there are plenty of code examples out there that aren't particularly brief, but are still useful. This example has repetition of common logic with simple differentiators to make it clear how to handle all of the basic verbs used for a REST API.

Reply:
Additionally, it has an example under the POST request showing how you can add business logic (even though it's simple logic).

Reply:
Actually, thinking about it more, you're right that this is too much to dump on at once.

I guess I interpreted the statement to mean that we shouldn't have the final product of the controller in one code snippet _at all_. Realistically, though, we could spend more time building up to that point, since we're probably not doing enough to explain what's happening step-by-step.